core/src/components/range/range.tsx

```diff
@@ -257,7 +267,10 @@ export class Range implements ComponentInterface {
     } else {
       this.ratioB = clamp(0, this.ratioB + step, 1);
     }
+
+    this.ionChangeStart.emit({ value: this.value });
```

Comment:
~ Consider using a local variable here, since we are accessing the same variable multiple times:
```tsx
const { value } = this;
this.ionChangeStart.emit({ value });
this.updateValue();
this.ionChangeEnd.emit({ value });
```

Reply:
`updateValue` will change the value, so we need to re-access it rather than using a cached version.

Reply:
Ahh I missed that, thanks for the clarification!